apps/web/src/app/layout.tsx

```diff
@@ -39,8 +37,18 @@ declare global {
   }
 }
 
+export default async function RootLayout({ children, params }) {
+  const isSandboxRoute = params?.sandboxId !== undefined
+
+  if (isSandboxRoute) {
+    return (
+      <html lang="en">
+        <head></head>
+        <body>{children}</body>
+      </html>
+    )
+  }
```

Comment:
Is this the correct way to do it?

Reply:
Shouldn't be there. I forgot to delete. Thanks for catching.